Code/GraphMol/RGroupDecomposition/RGroupScore.cpp

```diff
@@ -63,7 +74,7 @@ double matchScore(const std::vector<size_t> &permutation,
 #endif
       }
 #ifdef DEBUG
-      std::cerr << " " << rg->second->combinedMol->getNumAtoms(false)
+        std::cerr << l << " rgroup count" << num_rgroups_for_label << " num atoms" << rg->second->combinedMol->getNumAtoms(false)
                 // looks like code has been edited round this define
```

Comment:
I put in this comment as score is not defined.  If you don't need it perhaps delete both rows